Jenkinsfile

```diff
@@ -1,16 +1,8 @@
-def buildNumber = BUILD_NUMBER as int; if (buildNumber > 1) milestone(buildNumber - 1); milestone(buildNumber) // JENKINS-43353 / JENKINS-58625
 
-if (env.CHANGE_ID) {
-  properties([
-    buildDiscarder(logRotator(numToKeepStr: '10')),
-  ])
-} else {
-  properties([
-    buildDiscarder(logRotator(numToKeepStr: '96')),
- // Disable for now until we fix https://ci.jenkins.io/job/Infra/job/packer-images/job/master/148/console
- //   pipelineTriggers([[$class: "SCMTrigger", scmpoll_spec: "H/10 * * * *"]]),
-  ])
-}
+properties([
+  buildDiscarder(logRotator(numToKeepStr: '10')),
+  pipelineTriggers([cron('@daily')]),
```

Comment:
you want PRs to trigger this?

Reply:
yes, the intent is to execute this pipeline:

- On PRs (when a webhook is received from GitHub)
- Once a day

I've used the Pipeline syntax snippet generator to get this line: did I miss something?

Reply:
The `cron` trigger presumably makes sense only when `branch == 'main'`.

Reply:
@jglick good point ❤️